engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/runtime/task/IdentityLinkDto.ftl

```diff
@@ -0,0 +1,21 @@
+<@lib.dto required = ["type"] >
+
+    <@lib.property
+        name = "userId"
+        type = "string"
+        desc = "The id of the user participating in this link. Either `userId` or `groupId` is set." />
+
+    <@lib.property
+        name = "groupId"
+        type = "string"
+        desc = "The id of the group participating in this link. Either `groupId` or `userId` is set." />
+
+    <@lib.property
+        name = "type"
+        type = "string"
+        enumValues = ['"assignee"', '"candidate"', '"owner"']
```

Comment:
Maybe this is incorrect, but my understanding of the IdentityLink [documentation](https://github.com/camunda/camunda-bpm-platform/blob/master/engine/src/main/java/org/camunda/bpm/engine/task/IdentityLink.java#L38-L42) is that the type can be any String and those enum-values are just the ones the engine can handle by default.
If this is the case we would restrict this endpoint to only those three types.

Reply:
I agree it's technically possible to create additional, custom `Task`-`User` relationships, and we shouldn't restrict the values to the default three.

I will remove the `enumValues` and reference the engine-native values in the description.